language/de/extimgaslink.php

```diff
@@ -23,5 +23,11 @@
 }
 
 $lang = array_merge($lang, array(
+	'EXTIMGASLINK_CONFIG'			=> 'Zulässige Bilder',
+	'EXTIMGASLINK_CONFIG_EXPLAIN'	=> 'Bilder, die über eine gesicherte Verbindung verfügbar sind, können zusätzlich zugelassen werden. Alle übrigen Bilder werden durch Links ersetzt. Bilder, die unter der selben Domain wie dieses Board verfügbar sind, sind immer zugelassen.',
+	'EXTIMGASLINK_INVALID_CONFIG'	=> 'Die angegebene Bildherkunft ist ungültig.',
+	'EXTIMGASLINK_SECURE_SITES'		=> 'nur Bilder von sicheren Webseiten und dieser Domain',
```

Comment:
Da sollte es immer am Anfang groß geschrieben werden. In allen 6 betroffenen Zeilen.